packages/muelu/src/Interface/MueLu_FactoryFactory_decl.hpp

```diff
@@ -843,17 +843,17 @@ namespace MueLu {
     }
 #endif
 
-    RCP<FactoryBase> BuildBlockedDirectSolver(const Teuchos::ParameterList& /* paramList */, const FactoryMap& /* factoryMapIn */, const FactoryManagerMap& /* factoryManagersIn */) const {
-      //if (paramList.begin() == paramList.end())
+    RCP<FactoryBase> BuildBlockedDirectSolver(const Teuchos::ParameterList& paramList, const FactoryMap& /* factoryMapIn */, const FactoryManagerMap& /* factoryManagersIn */) const {
+      if (paramList.begin() == paramList.end())
```

Comment:
What about?
```suggestion
      if (paramList.numParas() == 0)
```
Or does the begin-end-comparison aim at something different?

Reply:
minor correction, `numParams()` right?